go.mod

```diff
@@ -33,7 +33,9 @@ require (
 	github.com/kr/text v0.1.0 // indirect
 	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/samber/lo v1.37.0 // indirect
```

Comment:
make it a direct dependency. `go mod add github.com/samber/lo; go mod tidy`

Reply:
guess you meant "go get"
anyway tidy did it